docs(route53): hostedZone import functions pre-requisites and usage causing confusion (#8830)
aws/aws-cdk: packages/@aws-cdk/aws-route53/README.md

Use the `CaaAmazonRecord` construct to easily restrict certificate authorities
allowed to issue certificates for a domain to Amazon only.

### Imports

> 

Reviewer: ```suggestion

Any Route53 Hosted Zone that has been created outside the stack can be imported into your CDK app
```

> If you don't know the ID of a Hosted Zone, you can use the `HostedZone.fromLookup`

Reviewer: ```suggestion
If you don't know the ID of the Hosted Zone to import, you can use the `HostedZone.fromLookup`
```

to discover and import it:

> Note that this requires an explicit environment to query an AWS account. Check
> out the [documentation](https://docs.aws.amazon.com/cdk/latest/guide/environments.html) for an overview of environments and how to implement.

Reviewer: perhaps we should include an example ourselves showing basic usage

```suggestion
⚠️ `fromLookup` requires an environment to be configured. Check
out the [documentation](https://docs.aws.amazon.com/cdk/latest/guide/environments.html) for an overview and examples
```

Author: yeah i think this is a good idea! i think something simple like 

```ts
new MyDevStack(app, 'dev', { 
  env: { 
    account: process.env.CDK_DEFAULT_ACCOUNT, 
    region: process.env.CDK_DEFAULT_REGION 
}});
```
and some description of using profiles should be sufficient, what do you think?
